core/conversion/var/Var.cpp

```diff
@@ -94,10 +94,18 @@ nvinfer1::ITensor* Var::ITensorOrFreeze(ConversionCtx* ctx) {
       "Requested either IValue containing a Tensor, or ITensor, however Var type is " << type_name());
 
   nvinfer1::ITensor* out;
-
+  auto weights = converters::Weights();
   if (isIValue()) {
-    auto weights = converters::Weights(ctx, ptr_.ivalue->toTensor());
-
+    auto tensor = ptr_.ivalue->toTensor();
```

Comment:
We should also check these cases if the setting is not enable to give users a hint to turn on truncating